src/db/schema.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  sqliteTable,
+  integer,
+  text,
+  foreignKey
+} from 'drizzle-orm/sqlite-core';
+
+// ---------------------------------------
+// TEACHER
+// ---------------------------------------
+export const teacher = sqliteTable('teacher', {
+  id: integer('id', { mode: 'number' }).primaryKey({ autoIncrement: true }),
+  first_name: text('first_name').notNull(),  // First name
+  last_name: text('last_name').notNull(),    // Last name
+  email: text('email').notNull(),            // Email
+  phone_number: text('phone_number'),        // Phone number
+  avatar: text('avatar'),                    // Avatar
+  priority: integer('priority')              // Priority
+});
+
+// ---------------------------------------
+// TEACHER_AVAILABILITY
+// ---------------------------------------
+export const teacherAvailability = sqliteTable('teacher_availability', {
+  id: integer('id', { mode: 'number' }).primaryKey({ autoIncrement: true }),
+  day: text('day').notNull(),                // Day (e.g. Monday, Tuesday...)
```

Comment:
Geht das als Enum?

Reply:
Es gibt keine Enums in SQLite, aber wir können das dann in den Modelklassen als Enum definieren.